src/allencell_ml_segmenter/main/main_model.py

```diff
@@ -32,6 +32,9 @@ def __init__(self) -> None:
         self._current_view: Optional[MainWindow] = None
         self._is_new_model: bool = False
         self.signals: MainModelSignals = MainModelSignals()
+        self._predictions_in_viewer: bool = (
+            False  # Tracks whether predictions are displayed in the viewer
```

Comment:
How is this info used?  Is this something about clearing images when the tab changes?